generators/app/templates/manifest.json

```diff
@@ -1,4 +1,12 @@
 {
   "name": "<%= pluginName %>",
-  "version": "0.1.0"
+  "displayName": "<%= displayName %>",
+  "description": "<%= description %>",
+  "license": "<%= license %>",
+  "version": "0.1.0",
+  "manifestVersion": "0.1.0",
```

Comment:
Do we want to use semver for the manifest version? I don't feel like this will change enough to warrant minor or patch updates?